CONTRIBUTING.md

```diff
@@ -124,6 +132,12 @@ development with the following:
   end
 ```
 
+## Granting Yourself Admin Access
+
+Once you've created an account with MyUSA, open the Rails console and change your admin status to "true". This will allow you to create and schedule messages.
+
+Note: scheduled messages may not send if you're using over the weekend, which is due to the business_time gem referenced above.
```

Comment:
WOW this is important. Wonder if we should be part of a separate bullet point so it is highlighted even more?